solutions/src/main/java/ru/sberbank/school/task02/service/FxConversionServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package ru.sberbank.school.task02.service;
+
+import ru.sberbank.school.task02.ExternalQuotesService;
+import ru.sberbank.school.task02.FxConversionService;
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+import ru.sberbank.school.task02.util.Volume;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public class FxConversionServiceImpl implements FxConversionService {
+
+    protected ExternalQuotesService externalQuotesService;
+
+    FxConversionServiceImpl(ExternalQuotesService externalQuotesService) {
+        this.externalQuotesService = externalQuotesService;
+    }
+    protected List<Quote> list;
```

Comment:
&#39;VARIABLE_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a396653485>*